NSS-Day-Cohort-53/tabloid-full-stack-always-picturesque-icebergs
# View All Categories Alphabetically

#### Changes Made
1. Added file `Category.cs` to `models` directory.
2. Added file `CategoryRepository.cs` to `repositories` directory.
3. Added file `ICategoryRepository.cs` to `repositories` directory.
4. Added file `CategoryController.cs` to `controllers` directory.
5. Added file `CategoryManager.js` to `client/modules` directory.
6. Added file `Categories.js` to `client/components` directory.
7. Modified file `applicationviews` to include `category route` functionality.

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout iw-getAllCategories
    ```
7. Open a new Terminal tab (⌘T) and navigate to the server directory.
8. Test app functionality.
    > On the nav bar select categories
    > When clicked a page of all categories ordered alphabetically should appear.
9. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
